packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -41,7 +41,13 @@ const BpkDataTable = (props: Props) => {
     children,
     className,
     defaultColumnSortIndex,
+    headerClassName,
+    headerHeight,
     height,
+    onRowClick,
+    rowClassName,
```

Comment:
Now this is explicitly called out should we add this to the README for the component?

Reply:
ah this was one of those props that we never explicitly supported, but was part of the API because of the library that we were using before and when I checked the usage some projects were using it